templates/react/{{cookiecutter.project_slug}}/{{cookiecutter.module_name}}/frontend/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "my_localstack_extension",
```

Comment:
here we should also parameterize from the cookiecutter parameters. perhaps `{{ cookiecutter.project_name }}` ?

Reply:
Yes, pardon :pray: